source/OAuth2Client.js

```diff
@@ -46,8 +46,9 @@ export default class OAuth2Client extends EventEmitter {
     }
 
     async exchangeAuthCodeForToken(authCode) {
+        const decodedAuthCode = Object.keys(parse(authCode))[0];
```

Comment:
Sorry, I must've skipped over this.. It's actually quite hard to read - Can we write this another way?

It looks as if you're parsing - getting an object - taking the keys and returning the first key?

Reply:
Yeah, it's a bit hard to read. Will update to be a non one-liner.

> It looks as if you're parsing - getting an object - taking the keys and returning the first key?

Correct.  The `parse` methods returns an object where the decoded value is the `key` sadly.